src/backend/data/post.js

```diff
@@ -146,7 +146,10 @@ class Post {
       article.guid,
       feed
     );
-    await Promise.all([post.save(), indexPost(post.text, post.id)]);
+    await Promise.all([
+      post.save(),
+      indexPost(post.text, post.id, post.title, post.updated, post.feed.author),
```

Comment:
Should we use `published` instead of `updated`? Both are useful, but I think `published` might make more sense?

cc @c3ho @humphd

Reply:
What's your thinking here?  Doesn't `updated` give us a more accurate sense of the timing of this post?

Reply:
Yes, you're right. I was thinking that maybe the original date was the one that we'd like to use, but `updated` makes more sense.